data.json

```diff
@@ -8,6 +8,15 @@
     "dateAdded": "2018-02-18T06:03:00.000Z",
     "tags": ["clothing", "device"]
   },
+  {
+    "name": "Alligator.io",
+    "difficulty": "hard",
+    "description": "Write a blog post on frontend development topics, get some cool Alligator stickers",
```

Comment:
Let's add some ending ponctuation to keep consistency with other swag items

Reply:
@aslafy-z whoops, wow good eye. I fixed it!